src/main/java/com/zontik/groshiky/controllers/AccountController.java

```diff
@@ -33,4 +37,14 @@ public ResponseEntity addAccount(Account account) {
         AccountModel newAccount = new AccountModel(account);
         return new ResponseEntity(newAccount, HttpStatus.OK);
     }
+
+    @GetMapping(value = "/transactions")
+    public ResponseEntity getTransactions (Integer accountId){
+        List<Transaction> transactionList = (accountService.findAccountById(accountId)).getTransactions();
+        List<TransactionModel> transactionModelsList = new ArrayList<TransactionModel>();
+        for (Transaction transaction:transactionList) {
+            transactionModelsList.add(new TransactionModel(transaction));
+        }
+        return new ResponseEntity(transactionModelsList, HttpStatus.OK);
```

Comment:
Using RestController you have no need to send response status now. You can return TransactionModel as it is and it'll be converted into json with appropriate status